q2_diversity_lib/tests/test_alpha.py

```diff
@@ -39,77 +39,47 @@ def test_non_phylogenetic_passed_empty_table(self):
-        self.empty_tree_as_NewickFormat = NewickFormat(empty_tree_fp, mode='r')
-        input_tree_fp = self.get_data_path('faith_test.tree')
-        self.input_tree_as_NewickFormat = NewickFormat(input_tree_fp, mode='r')
-        root_only_tree_fp = self.get_data_path('root_only.tree')
-        self.root_only_tree_as_NewickFormat = \
-            NewickFormat(root_only_tree_fp, mode='r')
-        missing_tip_tree_fp = self.get_data_path('missing_tip.tree')
-        self.missing_tip_tree_as_NewickFormat = \
-            NewickFormat(missing_tip_tree_fp, mode='r')
-
+        self.fn = self.plugin.actions['faith_pd']
+        self.tbl = self.artifact('FeatureTable[Frequency]',
+                                 'faith_test_table.biom')
+        self.tre = self.artifact('Phylogeny[Rooted]', 'faith_test.tree')
         self.expected = pd.Series({'S1': 0.5, 'S2': 0.7, 'S3': 1.0,
                                    'S4': 100.5, 'S5': 101},
                                   name='faith_pd')
 
     def test_receives_empty_table(self):
-        # empty table generated from self.empty_table with biom v2.1.7
-        empty_table = self.empty_table_as_BIOMV210Format
+        table = self.artifact('FeatureTable[Frequency]', 'empty_table.biom')
         with self.assertRaisesRegex(ValueError, 'empty'):
-            faith_pd(table=empty_table,
-                     phylogeny=self.input_tree_as_NewickFormat)
+            self.fn(table=table, phylogeny=self.tre)
 
     def test_method(self):
-        actual = faith_pd(table=self.input_table_as_BIOMV210Format,
-                          phylogeny=self.input_tree_as_NewickFormat)
-        pdt.assert_series_equal(actual, self.expected)
+        actual_art, = self.fn(table=self.tbl, phylogeny=self.tre)
+        self.assert_vec_equal(actual_art, self.expected)
 
     def test_accepted_types_have_consistent_behavior(self):
-        freq_table = self.input_table_as_BIOMV210Format
-        rel_freq_table = self.rf_table_as_BIOMV210Format
-        p_a_table = self.pa_table_as_BIOMV210Format
-        accepted_tables = [freq_table, rel_freq_table, p_a_table]
-        for table in accepted_tables:
-            actual = faith_pd(table=table,
-                              phylogeny=self.input_tree_as_NewickFormat)
-            pdt.assert_series_equal(actual, self.expected)
-
-    def test_passed_emptytree(self):
-        # NOTE: different regular expressions are used here and in
-        # test_beta.test_phylogenetic_measures_passed_emptytree() because
-        # Unifrac reports different error messages when an empty tree is passed
-        # to faith_pd than when the same is passed to the Unifrac methods.
-        with self.assertRaisesRegex(ValueError,
-                                    'table.*not.*completely represented'):
-            faith_pd(table=self.input_table_as_BIOMV210Format,
-                     phylogeny=self.empty_tree_as_NewickFormat)
-
-    def test_passed_rootonlytree(self):
```

Comment:
Same here

Reply:
Thanks, I think this was an accidental casualty - I'll reintroduce it.